product_pricelist_revision/models/pricelist.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Carlos Dauden - Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    name = fields.Char(search="_search_name")
```

Comment:
Instead of this, override `_rec_names_search`. Example:

https://github.com/OCA/l10n-spain/blob/0a689782d39ce264407d2143efba94b8c4e06f56/l10n_es_partner/models/res_partner.py#L40

Reply:
Done,

Reply:
This is not what I have said you. Please copy exactly the method and add the fields you want to search in.